spec/queue.js

```diff
@@ -0,0 +1,76 @@
+import chai, { expect, assert } from 'chai'
+import chaiChange from 'chai-change'
+import Queue from '../src/queue'
+
+chai.use(chaiChange)
+
+describe('Queue', () => {
+  'use strict'
+  let myQueue
+  beforeEach( () => {
+    myQueue = new Queue()
+  })
+
+  it('exists', () => {
+    expect(Queue).to.be.a('function')
+  })
+
+  context('#enqueue(element)', () => {
+    it('should add an element to the back of the queue', () => {
+      expect(() => myQueue.enqueue("foo")).to.alter(() => myQueue.length(), { from: 0, to: 1 })
+    })
+  })
+
+  context('#dequeue', () => {
+    it('should remove and return the front element in the queue', () => {
```

Comment:
Whenever I see "and" or "or" in a test description, I am suspicious that the test is trying to validate too many things.  I think the two tests here are: `it( 'should remove an element from the queue' )`, and `it( 'should remove the first element from the queue' )`.